src/components/FakeLink/FakeLink.tsx

```diff
@@ -19,6 +19,6 @@ export default class FakeLink extends React.Component<FakeLinkProps, {}> {
   render() {
     const { className, children } = this.props;
 
-    return <span className={classNames('y-fakeLink', className)}>{children}</span>;
+    return <span className={join(['y-fakeLink', className])}>{children}</span>;
```

Comment:
why not just use Array.prototype's join?

Reply:
ah I see, it is underneath. k

Reply:
this removes empty/undefined items so we don't have to check for the existence of `className` in each component